editor/src/components/inspector/inspector.tsx

```diff
@@ -425,6 +425,7 @@ export const Inspector = React.memo<InspectorProps>((props: InspectorProps) => {
         width: '100%',
         position: 'relative',
         height: '100%',
+        zIndex: 2,
```

Comment:
What's this for?

Reply:
the panels need to be above the cursors, which need to be above the canvas

canvas = 0
cursors = 1
panels = 2

(cc @balazsbajorics )

Reply:
Oooh, I hadn't spotted the one in the cursors. Could there be a little comment in each of those places saying why they are those values please?

Reply:
I don't like this. couldn't we put the cursors' div between the canvas and the panels in the dom to begin with?

Reply:
done in https://github.com/concrete-utopia/utopia/pull/4486/commits/3c5718f23bbc1d67a649913f4a6c9d3602103cda